.pre-commit-config.yaml

```diff
@@ -0,0 +1,33 @@
+# To use these pre-commit hooks you should first install https://pre-commit.com
+# and then run "pre-commit install" in the root folder of your local clone
+# so the hooks are deployed to .git/hooks
+#
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+#
+# the pep8 custom config (e.g. custom line lenght) is defined in file .pep8
+# https://pep8.readthedocs.io/en/release-1.7.x/intro.html#configuration
+
+repos:
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    sha: v1.2.0
+    hooks:
+      - id: autopep8-wrapper
+        name: 'autopep8-wrapper (.eb)'
+        types: [file]
+        files: \.eb$
+      - id: check-ast
+        name: 'check-ast (.eb)'
+        types: [file]
+        files: \.eb$
+      - id: autopep8-wrapper
+      - id: check-ast
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-merge-conflict
+
+  - repo: https://github.com/pescobar/pre-commit-hooks-easybuild
```

Comment:
@pescobar Wouldn't it be better to host this under `easybuilders`?

Reply:
@boegel fine for me

Reply:
OK, then you should transfer ownership of that repo to the `easybuilders` organisation, via https://github.com/pescobar/pre-commit-hooks-easybuild/settings

Reply:
@boegel I have transferred the repo to the easybuilders organization

I have granted access permissions to all the maintainers in easybuilders. Please let me know if you think the permissions should be adjusted